src/locale/locales/pt-BR/messages.po

```diff
@@ -7407,11 +7407,11 @@ msgstr "Dessilenciar thread"
 
 #: src/view/com/util/post-embeds/VideoEmbedInner/VideoWebControls.tsx:201
 msgid "Unmute video"
-msgstr ""
+msgstr "Dessilencia vídeo"
```

Comment:
In this context, "desmutar" is more appropriate than "dessilencia" as it is a recognized term in Portuguese and clearly conveys the action of resuming communication. "Dessilencia" is not a standard term and may cause confusion

Reply:
ok